pyproject.toml

```diff
@@ -50,15 +50,15 @@ black = ">=23.3.0"
 isort = ">=5.12.0"
 openpyxl = ">=3.0.10"
 tables = { version = ">=3.7.0" , python = "<4"}  # 3.8.0 depends on blosc2 which caps python to <4
-lxml = { version = ">=4.7.1,<4.9.0", python = "<3.11" }
+lxml = ">=4.7.1"
```

Comment:
```suggestion
lxml = ">=4.9.1"
```

The original issue was to try `lxml` to 4.9.1 due to some security vulnerability, so can we change that here?